src/react-components/ui-root.js

```diff
@@ -1067,13 +1067,18 @@ class UIRoot extends Component {
           ) : (
             <span>{this.props.hub.name}</span>
           )}
-          <button onClick={() => this.setState({ watching: true })} className={entryStyles.collapseButton}>
+          <button
+            aria-label="Close room entry modal"
```

Comment:
Would this be more descriptive as "Close room entry panel and enter lobby"?

Reply:
Yes